eng/AfterSigning.targets

```diff
@@ -0,0 +1,22 @@
+<Project>
+
+  <PropertyGroup>
+    <_SuppressSdkImports>false</_SuppressSdkImports>
+  </PropertyGroup>
+
+  <Target Name="PopulateGenerateChecksumItems"
+          AfterTargets="Build"
```

Comment:
@jander-msft  It failed to produce the hashed internally because the leg that does packaging and signing never calls the build target, unlike in ASP. Then neither this nor the `GenerateChecksums` targets will run. Adding a Target like this would solve the issue:

```xml
  <Target Name="GenerateSignedPackageChecksums"
        AfterTargets="Build;Pack"
        DependsOnTargets="PopulateGenerateChecksumItems;GenerateChecksums" />

  <Target Name="PopulateGenerateChecksumItems">
    <ItemGroup>
      <NuGetPackageFile Include="$(ArtifactsPackagesDir)**\*.nupkg" />
      <GenerateChecksumItems Include="%(NuGetPackageFile.Identity)" >
        <DestinationPath>%(FullPath).sha512</DestinationPath>
      </GenerateChecksumItems>
    </ItemGroup>
  </Target>
```

We could also add it as a publishing depends on in `eng\Publishing.props`.

Then for those to actually appear in the manifest we need to add them to the itemgroup. For an example on how to include them and give them the right metadata to publish https://github.com/dotnet/aspnetcore/blob/4a66e2d1ab32795c8f17776e488bce19ae43ff33/eng/Publishing.props#L53-L56